modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/fields-sidebar/components/FieldsSidebarSettingsBody.es.js

```diff
@@ -90,6 +89,7 @@ const getColumn = ({customFields = {}, ...otherProps}) => ({
 export default function ({
 	config,
 	customFields,
+	dataLayoutBuilder,
```

Comment:
I'm not a 100% sure that this property is not being used, because it's present on :
```javascript
getColumn({AppContext, customFields, dataLayoutBuilder}),
```
That is passing the value to a dynamic generated component:
```javascript
const CustomField = customFields[fieldName];

if (CustomField) {
	return (
		<div
			className="ddm-field"
			data-field-name={fieldName}
			key={index}
		>
			<CustomField
				{...otherProps}
				field={field}
				index={index}
			>
				{children}
			</CustomField>
		</div>
	);
}
```

Reply:
Now I got it! We will use the `dataLayoutBuilder` from `useContext(DataLayoutBuilderContext)`, right?

Reply:
Yeah, this is probably being used in App Builder, maybe we can do what I said in another comment https://github.com/liferay-forms/liferay-portal/pull/128#discussion_r574504781